lib/WooCommerce/Settings.php

```diff
@@ -2,8 +2,58 @@
 
 namespace MailPoet\WooCommerce;
 
+use MailPoet\Settings\SettingsController;
+
 class Settings {
+
+  /** @var SettingsController */
+  private $settings;
+
+  function __construct(SettingsController $settings) {
+    $this->settings = $settings;
+  }
+
   function disableWooCommerceSettings() {
-    // TODO
+    if ($_GET['tab'] !== 'email') {
+      return;
+    }
+    if (!(bool)$this->settings->get('woocommerce.use_mailpoet_editor')) {
+      return;
+    }
+    $woocommerce_template_id = $this->settings->get('woocommerce.transactional_email_id');
+    ?>
+
```

Comment:
Is it possible to put the CSS and HTML code inside a Twig view and render it instead of mixing everything here?

Reply:
Great suggestion, moved to Twig view. Also fixed checking undefined array items.